packages/flutter/lib/src/material/checkbox_list_tile.dart

```diff
@@ -320,6 +321,9 @@ class CheckboxListTile extends StatelessWidget {
   /// Defaults to Color(0xFFFFFFFF).
   final Color? checkColor;
 
+  /// {@macro flutter.material.ListTile.tileColor}
+  final Color tileColor;
```

Comment:
Shouldn't null be an allowable value here to match the rest of the properties? If `CheckboxListTile` is null, then it should use the theme value or some default value, right?

```suggestion
  final Color? tileColor;
```

Reply:
Oh yeah, seems like this is the reason for the failing checks: https://github.com/flutter/flutter/pull/67135/checks?check_run_id=1232817051

Reply:
Thanks, didin't noticed after I merged NNBD changes.